src/pipeline/bundle-js.js

```diff
@@ -31,12 +41,29 @@ module.exports = function (opts, paths) {
       cacheFile: path.join(paths.TMP_DIR, 'browserify-cache.json'),
       transform: getTransform(opts, paths),
       plugin: [
-        (b) => b.require([
-          {file: paths.SYNTAX_OUTPUT_FILE, expose: '__IDYLL_SYNTAX_HIGHLIGHT__'},
-          {file: paths.AST_OUTPUT_FILE, expose: '__IDYLL_AST__'},
-          {file: paths.COMPONENTS_OUTPUT_FILE, expose: '__IDYLL_COMPONENTS__'},
-          {file: paths.DATA_OUTPUT_FILE, expose: '__IDYLL_DATA__'}
-        ])
+        (b) => {
+          const requires = [{
+            key: 'syntaxHighlighting',
+            requireName: '__IDYLL_SYNTAX_HIGHLIGHT__'
+          }, {
+            key: 'data',
+            requireName: '__IDYLL_DATA__'
+          }, {
+            key: 'components',
+            requireName: '__IDYLL_COMPONENTS__'
+          }, {
+            key: 'ast',
+            requireName: '__IDYLL_AST__'
+          }];
+          requires.forEach((obj) => {
+            b.exclude(obj.requireName);
+
+            b.require(toStream(output[obj.key]), {
```

Comment:
there's probably a way to create the input as streams directly instead of creating strings and converting, but this seems fine for the moment